src/modules/my-positions/actions/load-account-positions.js

```diff
@@ -1,14 +1,28 @@
+import async from 'async'
 import { augur } from 'services/augurjs'
 import { updateAccountPositionsData } from 'modules/my-positions/actions/update-account-trades-data'
 import logError from 'utils/log-error'
+import { loadMarketsInfo } from 'modules/markets/actions/load-markets-info'
 
 export const loadAccountPositions = (options, callback = logError) => (dispatch, getState) => {
   const { universe, loginAccount } = getState()
   if (!loginAccount.address) return callback(null)
   augur.trading.getUserTradingPositions({ ...options, account: loginAccount.address, universe: universe.id }, (err, shareBalances) => {
     if (err) return callback(err)
     if (shareBalances == null) return callback(null)
-    dispatch(updateAccountPositionsData(shareBalances, options.marketID))
-    callback(null, shareBalances)
+    // TODO: need better way to consolidate by marketID
+    const marketIDs = shareBalances.reduce((collapsed, position) => {
```

Comment:
The convention we’ve adopted in the CB for reduce args is `(p, <item>, …)`.

Reply:
Is this what you mean? is `position` considered a <singular item name> ?
    
const marketIDs = positions.reduce((p, position) => {
      if (!p[position.marketID]) {
        p[position.marketID] = []
      }
      p[position.marketID].push(position)
      return p
    }, {})

Reply:
Yup, that’s what I was driving towards.

I’m going to do a slight cleanup here to simplify this method, give me one sec.

Reply:
K, pushed!